shimTool/guiUtils.py

```diff
@@ -49,16 +141,37 @@ def stop(self):
 
 
 class ImageViewer(QGraphicsView):
-    def __init__(self, parent=None, label=None):
+    def __init__(self, parent=None, layout=None):
         super(ImageViewer, self).__init__(parent)
         self.scene = QGraphicsScene(self)
         self.setScene(self.scene)
         self.pixmap_item = None
         self.qImage: QImage = None
         self.viewData = None  # 2D data that the image viewer is currently being set to show
-        self.label = label
+        self.label = QLabel()
+        self.width = 512
+        self.height = 512
+        self.setFixedSize(self.width, self.height)  # Set a fixed size for the view
+        self.setRenderHints(QPainter.RenderHint.Antialiasing | QPainter.RenderHint.SmoothPixmapTransform)
+        self.colorbar = ColorBar()
+        self.colorbar.setFixedWidth(110)  # Set a fixed width for the color bar
+        self.colorbar.setFixedHeight(self.height)  # Match height with the viewer
+
+        # Layout to include image and color bar
+        # Not sure if I should put the part below in Gui.py or here, like most of the add widget functions have been in gui.py
```

Comment:
can remove this line too since i think we agree that it is better having it here. more clean in gui.py